checkpoint3/TodoProgram.cs

```diff
@@ -0,0 +1,461 @@
+﻿// https://youtu.be/ciNHn38EyRc
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Data.Sqlite;
+
+namespace TodoProgram
```

Comment:
Nice job overall, but your methods are doing more than they should.
It would be nice if there was a separation between the methods that got the data from the database, and when the printing to the screen is done.